### PR TITLE
Addressing issue #58 with guidance for handling non-DDR queries for resolver.arpa

### PR DESCRIPTION
**Pull request by tojens-ietf**
Addressing #58

### CODE REVIEW

**Review by tfpauly — APPROVED** [review]
@bemasc can you take a look too?

File: draft-ietf-add-ddr.md [hunk 1]
```diff
@@ -391,6 +391,14 @@ resolvers that support discovery using IP addresses will need to be
 configured to present the appropriate TLS certificate when no SNI is present
 for both DoT and DoH.
 
+## Handling non-DDR queries for resolver.arpa
+
+DNS resolvers that support DDR by responding to queries for _dns.resolver.arpa
+SHOULD treat all domain names under resolver.arpa as locally served per {{?RFC6303}}.
```
Comment (bemasc) [inline draft-ietf-add-ddr.md:397]:
```suggestion
SHOULD treat resolver.arpa as a locally served zone per {{!RFC6303}}.
```

**Review by bemasc — COMMENTED** [review]
I think the draft also needs to update [the registry](https://www.iana.org/assignments/locally-served-dns-zones/locally-served-dns-zones.xhtml#transport-independent)

**Comment by mstojens** [comment]
@bemasc I've bravely stolen some text to meet the additional IANA request.

**Review by bemasc — APPROVED** [review]

**Review by Magkoooh — APPROVED** [review]

**Pull request merged**